swimlane/core/resources/usergroup.py

```diff
@@ -119,3 +133,23 @@ def get_cache_index_keys(self):
             'username': self.username,
             'display_name': self.display_name
         }
+
+
+class GroupUsersCursor(SwimlaneResolver, Cursor):
+    """Handles retrieval for user endpoint"""
+
+    def __init__(self, swimlane, user_ids=None):
```

Comment:
Don't default to `None` since this should be a required parameter for the cursor to work correctly.